connectors-home/modules/ROOT/partials/verbose-logs.adoc

```diff
@@ -0,0 +1,26 @@
+// Used in Connector troubleshooting pages.
+== Enable Verbose Logging
+
+To get a better understanding of why an application's interaction with the {page-service-name} Connector is failing, temporarily enable verbose logging for the connector. +
```

Comment:
Is the {page-service-name} a variable? Should it be changed to the connector name?

Reply:
It's a variable!